src/main/java/com/blogify/blogapi/repository/model/User.java

```diff
@@ -0,0 +1,44 @@
+package com.blogify.blogapi.repository.model;
+
+import static javax.persistence.GenerationType.IDENTITY;
+import java.io.Serializable;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "\"user\"")
+public class User implements Serializable {
+  @Id
+  @GeneratedValue(strategy = IDENTITY)
```

Comment:
front will give the ID, so d'ont use @GeneratedValue

Reply:
okay